src/wymeditor/editor/undo-redo.js

```diff
@@ -88,6 +88,7 @@ WYMeditor.UndoRedo.prototype._add = function () {
         wym = undoRedo.wym;
 
     undoRedo.history.add(wym.getCurrentState());
+    undoRedo.hasUnregisteredModification = false;
```

Comment:
@winhamwr I changed the name from `hasUnregisteredEdits` to `hasUnregisteredModification`.

This name is more suitable because of the existing `registerModification` method.